020_app_L44/020_app_L44_unsafe.py

```
@@ -0,0 +1,108 @@
+
+
+@app.route("/pt/<string:payload>")
+def definite_pt(payload: str) -> str:
+    f = open(payload, "r")
+# {fact rule=cross-site-scripting@v1.0 defects=1}
+    text = f.read()
+    return text
+
+@app.route("/xss/<string:payload>")
+def definite_xss(payload: str) -> None:
+    content = flask.Markup(payload)
```

Ruff (0.14.1)

Unsafe use of `flask.Markup` detected

(S704)